### PR TITLE
docs: minimal contributing documentation

### PR DESCRIPTION
**Pull request by wtrocki**
Kickstarting some of the documentation by explaining patterns we use in the Atlas CLi. 
This is by no means complete but it does provide minimal level of information that one needs to get started.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -132,12 +132,28 @@ Review and replace command name and arguments depending on the command you are u
 
 ```
 
+### Contributing New Feature
 
-### API Interactions
+`Atlas CLI` and `MongoDB CLI` using[Cobra Framework](https://umarcor.github.io/cobra/) framework.
```
Comment (gssbzn) [inline CONTRIBUTING.md:137]: should this be combined with https://github.com/mongodb/mongodb-atlas-cli/blob/cb185740535b25ee542c41c270b23d15ab405366/CONTRIBUTING.md#adding-a-new-command it seems to say many of the same things but with different words

Comment (wtrocki) [reply CONTRIBUTING.md:137]: Fixed

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -132,17 +132,32 @@ Review and replace command name and arguments depending on the command you are u
...
```
Comment (gssbzn) [inline CONTRIBUTING.md:137]:
```suggestion
`Atlas CLI` and `MongoDB CLI` are using the [Cobra Framework](https://umarcor.github.io/cobra/).
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -132,17 +132,32 @@ Review and replace command name and arguments depending on the command you are u
 
 ```
 
+### Contributing New Feature
```
Comment (gssbzn) [inline CONTRIBUTING.md:135]: can you please update the toc

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -157,6 +172,12 @@ For commands that create or modify complex data structures, the use of configura
 
 Note: we are experimenting with a generator, make sure to try it out in [tools/cli-generator](./tools/cli-generator/)
 
+### API Interactions
+
+Atlas CLI and MongoDB CLI use [go-client-mongodb-atlas](https://github.com/mongodb/go-client-mongodb-atlas/)
```
Comment (gssbzn) [inline CONTRIBUTING.md:177]: this may be stale now, go-sdk?

**Review by gssbzn — COMMENTED** [review]
I think it may be worth extracting new features to a new doc which includes new comamnd, new feature is quite vague but the current description seems to assume a new feature requires a new root command, also with the current edits it doesn't fully address that new feature and add a new command are kind of the same, new feature seems to be add a new root command, while add a new command is  more generic and vague (not necessary better)

File: CONTRIBUTING.md (hunk 4 above)
Comment (wtrocki) [reply CONTRIBUTING.md:177]: Good point! will extend it to cover both SDKs.

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -132,17 +132,32 @@ Review and replace command name and arguments depending on the command you are u
...
 
+### Contributing New Feature
 
-### API Interactions
+`Atlas CLI` and `MongoDB CLI` using[Cobra Framework](https://umarcor.github.io/cobra/) framework.
 
-Atlas CLI and MongoDB CLI use [go-client-mongodb-atlas](https://github.com/mongodb/go-client-mongodb-atlas/) 
-and [go-client-mongodb-ops-manager](https://github.com/mongodb/go-client-mongodb-ops-manager/) to interact with Atlas or Ops Manager/Cloud Manager.
-Any new feature should first update the respective client.
+Each new feature can be introduced by creating new cobra root command.
```
[the remark this answers is not in the document]
Comment (wtrocki) [reply CONTRIBUTING.md]: Clarified that relationship. It was indeed confusing

**Review by gssbzn — APPROVED** [review]
LGTM

**Comment by wtrocki** [comment]
Thank you @gssbzn

**Pull request merged**